src/main/java/com/alesya/mafa/pet/Cat.java

```diff
@@ -23,7 +24,28 @@ public void setHair(String hair) {
         this.hair = hair;
     }
 
+    public enum Breed {
+        ABYSSINIAN, BENGAL, BIRMAN, BOMBAY, BURMESE, CORNISH_REX
+    }
```

Comment:
Please add for feature task - add nice printing of these breeds

Reply:
done